src/test/scala/nl.knaw.dans.easy.multideposit/TestSupportFixture.scala

```diff
@@ -93,6 +94,7 @@ trait TestSupportFixture extends FlatSpec with Matchers with OptionValues with I
         audiences = List("D37000"),
         accessright = AccessCategory.GROUP_ACCESS
       ),
+      baseUUID = "ab23-bdgg",
```

Comment:
this is not a valid UUID and this example is supposed to be valid

Reply:
replaced it with a valid one an created a new testInstructions with invalid uuid